src/commands/profile.js

```diff
@@ -5,15 +5,40 @@ const colors = require('colors/safe');
 const Logger = require('../logger.js');
 const User = require('../models/user.js');
 
-async function profile () {
-  const { id, name, email, preferredMFA } = await User.getCurrent();
-  const has2FA = (preferredMFA != null && preferredMFA !== 'NONE') ? 'yes' : 'no';
-  const formattedName = name || colors.red.bold('[not specified]');
-  Logger.println('You\'re currently logged in as:');
-  Logger.println('User id          ' + id);
-  Logger.println('Name             ' + formattedName);
-  Logger.println('Email            ' + email);
-  Logger.println('Two factor auth  ' + has2FA);
+async function profile (params) {
+  const { format } = params.options;
+
+  const user = await User.getCurrent();
+
+  const formattedUser = {
+    id: user.id,
+    email: user.email,
+    name: user.name,
+    phone: user.phone,
+    address: user.address,
+    city: user.city,
+    zipcode: user.zipcode,
+    country: user.country,
+    creationDate: new Date(user.creationDate),
+    lang: user.lang,
+    emailValidated: user.emailValidated,
+    has2FA: (user.preferredMFA != null && user.preferredMFA !== 'NONE'),
+  };
+
+  switch (format) {
+    case 'json': {
+      Logger.printJson(formattedUser);
+      break;
+    }
```

Comment:
Maybe we could just reply with full user data ?

```suggestion
  switch (format) {
    case 'json': {
      Logger.printJson(user);
      break;
    }
```

Reply:
So maybe we should add `avatar`, `oAuthApps`, `admin` and send same infos in both formats as mentioned below

Reply:
Let's go for :

```
id: user.id,
email: user.email,
name: user.name,
avatar: user.avatar,
creationDate: new Date(user.creationDate),
lang: user.lang,
has2FA: (user.preferredMFA != null && user.preferredMFA !== 'NONE'),
```